conda/install.py

```diff
@@ -575,7 +575,8 @@ def read_no_link(info_dir):
 # Should this be an API function?
 def symlink_conda(prefix, root_dir, shell=None):
     # do not symlink root env - this clobbers activate incorrectly.
-    if normpath(prefix) == normpath(sys.prefix):
+    # prefix should always be longer than, or outside the root dir.
+    if normpath(prefix) in normpath(root_dir):
```

Comment:
This comparison does not work properly on my win7 system and I still see my activate and deactivate scripts being overwritten.  I believe this is due to case sensitivities in the comparison.  The activate script is called with prefix=d:\Anaconda3-64b but root_dir is D:\Anaconda3-64b.  Wrapping the calls to normpath with normcase fixes the issue:
`    if normcase(normpath(prefix)) in normcase(normpath(root_dir)):`